test/functional/search_api_test.rb

```diff
@@ -0,0 +1,36 @@
+require 'test_helper'
+
+class SearchApiTest < ActiveSupport::TestCase
+  include Rack::Test::Methods
+
+  def app
+    Rails.application
+  end
+
+  test 'search notes functionality' do
+    get '/api/srch/notes?srchString=Blog'
+    assert last_response.ok?
+
+    # Expected search pattern
+    pattern = {
+      srchParams: {
+        srchString: 'Blog',
+        seq: nil,
+      }.ignore_extra_keys!
+    }.ignore_extra_keys!
+
+    matcher = JsonExpressions::Matcher.new(pattern)
+
+    puts JSON.parse(last_response.body)
+    assert matcher =~ JSON.parse(last_response.body)
+    
+
+    # check for actual Blog note, too
```

Comment:
OK, great -- this looks super and it's great to see the test passing now. Can you remove these comments, and add a few more items to the above search pattern, based on the output that the `puts` generated? (i copied this into an earlier comment, so you can work from there). 

Because at the moment, we don't actually test anything about the returned results -- see how the pattern is just a confirmation of the /request/ we made, not the returned search?